### PR TITLE
implement client-only layout.

### PR DESCRIPTION
**Pull request by NiklasRentzCAU**
Hardcode to use it for now, switch between modes missing yet.

### CODE REVIEW

**Review by Eddykasp — COMMENTED** [review]
We should import all of elks algorithms not just layered

**Comment by NiklasRentzCAU** [comment]
Before merging, this PR needs to include a way to switch between server and client layout, with the default being server layout for now.
For further improvement, this PR could also include a new message part in the "capabilities" messaging that asks the server if it supports server layout (to which the Java server would reply true) and automatically adapt its setting to the supported one (client layout if the server does not support it, server layout otherwise)

**Comment by Eddykasp** [comment]
- [ ] add json schema for our lsp messages
- [ ] add validification using the new json schemas

**Comment by NiklasRentzCAU** [comment]
Closes #139

**Review by Eddykasp — COMMENTED** [review]
I think this is still missing the change to make the debug option switch no longer a debug option

File: packages/klighd-core/src/options/render-options-registry.ts [hunk 1]
```diff
@@ -509,8 +509,6 @@ export class DebugOptions implements RenderOption {
     readonly description = 'Whether debug options should be shown.'
 
     currentValue = false
-
-    debug = true
```
Comment (NiklasRentzCAU) [inline packages/klighd-core/src/options/render-options-registry.ts:513]: "DebugOptions" is no longer a debug option, see here.

**Review by Eddykasp — APPROVED** [review]
lgtm

**Pull request merged**